Source/WebCore/page/FocusController.cpp

```diff
@@ -361,15 +361,17 @@ void FocusController::setFocusedFrame(Frame* frame)
 
     // Now that the frame is updated, fire events and update the selection focused states of both frames.
     if (auto* oldFrameView = oldFrame ? oldFrame->view() : nullptr) {
-        oldFrameView->stopKeyboardScrollAnimation();
+        oldFrameView->stopKeyboardScrollAnimation([] {
```

Comment:
Smush onto one line (or maybe make it take optional/null/something so we don't have to pass an empty block?)

Reply:
should the lines below this actually go into this completion block instead? Or is it fine as-is?